src/pages/partners.tsx

```diff
@@ -81,21 +81,21 @@ export default function partners(): ReactElement {
 				<ContainerInside>
 					<Heading mb={10}>Our Partners</Heading>
 					<Tabs variant="line" colorScheme="purple" align="center">
-						<Box py={2} backgroundColor="#D8D6EC">
+						<Box backgroundColor="#D8D6EC">
 							<TabList
 								justify="center"
 								align="center"
 								color="#8287BE"
 							>
 								<Tab
-									_selected={{ textDecoration: "underline" }}
-									fontWeight={100}
-									py={3}
+									_selected={{
+										borderBottom: "5px solid #8287BE",
```

Comment:
I don't like the "5px" here, but to circumvent that, you'd have to write out the `border` property. Can be fixed later.